app/javascript/packs/actions/initialState.js

```diff
@@ -77,12 +77,12 @@ const formatState = {
 };
 
 export function loadInitialState(initialState) {
-  const types = Object.keys(initialState);
   const formatters = Object.keys(formatState);
-
-  let formattedState = types.filter((type) => {
+  const types = Object.keys(initialState).filter((type) => {
```

Comment:
😅Thank you

Reply:
oh wait i do not remember touching this is this a bad or a good?